dashboard-api/docker-compose-dev.yaml

```diff
@@ -0,0 +1,17 @@
+services:
+  api:
+    build:
+      context: .
+      dockerfile: Dockerfile.dev
+    environment:
+      STUDIESMANAGER_HOST: 0.0.0.0
+      STUDIESMANAGER_PORT: 80
+      STUDIESMANAGER_DBURI: postgresql://root@cockroachdb:26257/vlab?sslmode=disable
```

Comment:
For consistency reasons, shouldn't we use the same env var declarations everywhere? Currently, in `system` we use https://github.com/vlab-research/vlab/pull/75/files#diff-fc992c9a9d4e6e1728cac26d958fcbe0cde48518b65c36fbd6740aa47d85b8a3R30-R33.

Reply:
Agree, will update the names to be more aligned 👍

Reply:
Sorry for the confusion, but what I meant to say is that instead of declaring a single URI string for the DB, we split it into multiple env vars. 

Instead of this:
```
STUDIESMANAGER_DBURI: postgresql://root@cockroachdb:26257/vlab?sslmode=disable
```

we do this:
```
DATABASE_NAME: vlab
DATABASE_HOST: cockroachdb
DATABASE_PORT: 26257
DATABASE_USER: root
```

This will be more consistent with how we define DBs in `system`, see https://github.com/vlab-research/vlab/pull/75/files#diff-fc992c9a9d4e6e1728cac26d958fcbe0cde48518b65c36fbd6740aa47d85b8a3R30-R33.